### PR TITLE
@rahulsain3000 sir i have submitted day_22 of the series at Hackerrank_30daysOFcode so please review my work and add it .

### PR DESCRIPTION
**Pull request by Utsavrai778**

### CODE REVIEW

**Review by rahulsain3000 — CHANGES_REQUESTED** [review]
Please also mark ✅  on readme file too

**Comment by rahulsain3000** [comment]
Please resolve conflicts before making any change to readme file

**Comment by Utsavrai778** [comment]
sir please check once again readme file as i have also marked check in readme file still i am getting one conflict can you resolve this issue.

File: 30daysOFcode_Swift/day_22.swift [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        if let currentRoot = root {
+            let leftHeight = getHeight(root: currentRoot.left)
+            let rightHeight = getHeight(root: currentRoot.right)
+            
+            return 1 + max(leftHeight, rightHeight)
+        }
+        
+        return -1
+    }
+
```
Comment (Utsavrai778) [inline 30daysOFcode_Swift/day_22.swift:38]: this is the required function to be coded there and hence i have completed this function portion.

**Comment by Utsavrai778** [comment]
sir after many attempt i am unable to resolve that conflict can please help me to get of this because i already  have noticed that in that file there is extra '|' symbol so please can u delete it if the same is creatng  that error .

**Pull request merged**